config-xml/addon/src/main/java/org/jboss/windup/config/parser/metadata/MetadataDependencyAddon.java

```diff
@@ -0,0 +1,30 @@
+package org.jboss.windup.config.parser.metadata;
+
+import org.apache.commons.lang.StringUtils;
+import org.jboss.forge.furnace.addons.AddonId;
+import org.jboss.windup.config.exception.ConfigurationException;
+import org.jboss.windup.config.parser.ElementHandler;
+import org.jboss.windup.config.parser.NamespaceElementHandler;
+import org.jboss.windup.config.parser.ParserContext;
+import org.w3c.dom.Element;
+
+@NamespaceElementHandler(elementName = "addon", namespace = "http://windup.jboss.org/v1/xml")
+public class MetadataDependencyAddon implements ElementHandler<AddonId>
+{
+    private static String ID="id";
+    @Override
+    public AddonId processElement(ParserContext handlerManager, Element element) throws ConfigurationException
+    {
+        String id = element.getAttribute(ID);
+        if (!StringUtils.isBlank(id))
```

Comment:
Need to throw a validation exception if this value is missing. (Like our existing handlers.)